### DISCUSSION TITLE
Subreddit for Authelia

### DISCUSSION THREAD

**Discussion by guettli** [authelia/authelia#7491] [Polls]
### Description

I don't use matrix or discord.

I think a Subreddit for Authelia would be great

### Use Case

Ask questions, learn by reading questions from other users.

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)

- [X] I have checked for related issues and checked the documentation

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

Currently, Authelia does not have an official subreddit. However, there are several other ways to engage with the community and get support:

1. **GitHub Discussions**: You can discuss anything that is not a bug or feature request, such as ideas about future features, questions, support requests, and sharing configuration or utilization ideas. [GitHub Discussions](https://github.com/authelia/authelia/discussions)

2. **GitHub Issues**: If you have found a bug or have a feature request, you can lodge it here after checking that it has not already been reported. [GitHub Issues](https://github.com/authelia/authelia/issues)

3. **Community Chat**: Authelia has a community chat service accessible via [Matrix](https://matrix.to/#/#community:authelia.com) or [Discord](https://discord.authelia.com).

4. **Email**: For non-security related matters, you can contact the team via [team@authelia.com](mailto:team@authelia.com). For security-related matters, use [security@authelia.com](mailto:security@authelia.com).

If you have any specific questions or need support, these platforms should be able to assist you effectively.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

Currently, Authelia does not have an official subreddit. However, there are several other ways to engage with the community and get support:

1. **GitHub Discussions**: You can ask questions, share ideas, and get support from the community on the [GitHub Discussions](https://github.com/authelia/authelia/discussions) forum.

2. **Matrix and Discord**: Authelia has community chat services on both [Matrix](https://matrix.to/#/#community:authelia.com) and [Discord](https://discord.authelia.com). These platforms have various rooms and channels for support, contributing, and off-topic discussions.

3. **Email**: For non-security related matters, you can contact the team via email at [team@authelia.com](mailto:team@authelia.com).

If you have any specific questions or need support, these platforms are great places to start.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by james-d-elliott** [comment]
We have reserved the authelia subreddit. What would be the advantage over the GitHub Discussions area (which I moved your issue to)?

**Reply by guettli** [reply]
Github discussion are mostly like this: New users ask the maintainers questions.

Reddit is more peer-to-peer (if there is a healthy community).

I scroll through Reddit from time to time in my spare-time. I don't do that for github discussions.
